torch/onnx/symbolic_helper.py

```diff
@@ -536,32 +536,19 @@ def _slice_helper(g, input, axes, starts, ends, steps=None, dynamic_slice=False)
 
 
 def _is_fp(value):
+    torch_types = (torch.float16, torch.float32, torch.float64, torch.bfloat16)
```

Comment:
```suggestion
    torch_types = {torch.float16, torch.float32, torch.float64, torch.bfloat16}
```

nit: use a set for constant look up

Reply:
Is there a reason for not creating a set directly? set([torch.float16, torch.float32, torch.float64, torch.bfloat16]) creates a list which is just another extra step. As a toy example

```
>>> timeit.timeit('{str, int, float, dict, set}', number=100000)
0.01923950000127661
>>> timeit.timeit('set([str, int, float, dict, set])', number=100000)
0.03770090000034543
```